purchase_sale_inter_company/models/stock_picking.py

```diff
@@ -21,19 +21,42 @@ def _action_done(self):
             purchase.picking_ids.write({"intercompany_picking_id": pick.id})
             if not pick.intercompany_picking_id and purchase.picking_ids[0]:
                 pick.write({"intercompany_picking_id": purchase.picking_ids[0]})
-            for move_line in pick.move_line_ids:
-                sale_line_id = move_line.move_id.sale_line_id
-                po_move_lines = sale_line_id.auto_purchase_line_id.move_ids.mapped(
+            for move in pick.move_lines:
```

Comment:
Why is the variable name `move`? I think the variable name should be `line` because you are iterating over the lines.